### ISSUE TITLE
Crashing when open

### ISSUE THREAD

**Issue by xmha97** [LibChecker/LibChecker#598]
### LibChecker version / LibChecker 版本

2.3.6.e69c5d44

### Bug description / Bug 描述

https://user-images.githubusercontent.com/23485114/222008600-041be148-177a-40e8-8a56-37f2de771d32.mp4

### Expected behavior / 预期行为

Open app

### Actual behavior / 实际行为

Crashed.

### Steps to reproduce / 复现步骤

1. Open the app.
2. Crashed.

### UI / OS

MIUI 14

### Android Version / Android 版本

13

### Additional info / 额外信息

```
java.lang.IllegalStateException: Unable to parse composition
	at com.absinthe.libchecker.hq0.onResult(SourceFile:51)
	at com.airbnb.lottie.LottieAnimationView$a.onResult(SourceFile:18)
	at com.absinthe.libchecker.bo.run(SourceFile:162)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:211)
	at android.os.Looper.loop(Looper.java:300)
	at com.absinthe.libchecker.b90.run(SourceFile:1)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:211)
	at android.os.Looper.loop(Looper.java:300)
	at android.app.ActivityThread.main(ActivityThread.java:8143)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1028)
Caused by: java.util.zip.ZipException: invalid entry size (expected 202244 but got 202066 bytes)
	at java.util.zip.ZipInputStream.readEnd(ZipInputStream.java:421)
	at java.util.zip.ZipInputStream.read(ZipInputStream.java:211)
	at java.util.zip.ZipInputStream.closeEntry(ZipInputStream.java:148)
	at java.util.zip.ZipInputStream.getNextEntry(ZipInputStream.java:121)
	at com.absinthe.libchecker.pq0.g(SourceFile:140)
	at com.absinthe.libchecker.pq0.f(SourceFile:1)
	at com.absinthe.libchecker.pq0.b(SourceFile:44)
	at com.absinthe.libchecker.lq0.call(SourceFile:332)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
	at java.lang.Thread.run(Thread.java:1012)
```

**Comment by zym013579** [comment]
应该跟时间有关系，我把系统时间设置到3月1日之前（不含3月1日），就可以正常打开了

This may be caused by the date. It works well when I set the date before March 1st (excluding March 1st).

**Comment by WanderMax** [comment]
我也出现了这个

**Comment by swift-fs** [comment]
手机系统版本Android 13.0，尝试了2.3.2至2.3.6的所有版本，都是打开就立即闪退。

The mobile phone system version is Android 13.0, tried all versions from 2.3.2 to 2.3.6, and it crashed immediately after opening

**Comment by lbydev** [comment]
It seems that there is a problem with the spring animation resource package, `anim/lib_reference_spring.json.zip`,waiting for the author to fix!

**Comment by xmha97** [comment]
> 手机系统版本Android 13.0，尝试了2.3.2至2.3.6的所有版本，都是打开就立即闪退。
> 
> The mobile phone system version is Android 13.0, tried all versions from 2.3.2 to 2.3.6, and it crashed immediately after opening

Yes, my phone is Android 13 (not 12)

**Comment by xmha97** [comment]
v2.3.7 open but statistics tab not working...


https://user-images.githubusercontent.com/23485114/222029145-9fc4c6dd-f583-4b8c-8b84-cb28e428110a.mp4

**Comment by xmha97** [comment]
# Log (v2.3.7)
```
java.lang.IllegalStateException: Unable to parse composition
	at com.absinthe.libchecker.sq0.onResult(SourceFile:51)
	at com.airbnb.lottie.LottieAnimationView$a.onResult(SourceFile:18)
	at com.absinthe.libchecker.do.run(SourceFile:162)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:211)
	at android.os.Looper.loop(Looper.java:300)
	at com.absinthe.libchecker.f90.run(SourceFile:1)
	at android.os.Handler.handleCallback(Handler.java:942)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:211)
	at android.os.Looper.loop(Looper.java:300)
	at android.app.ActivityThread.main(ActivityThread.java:8143)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1028)
Caused by: java.util.zip.ZipException: invalid entry size (expected 202244 but got 202066 bytes)
	at java.util.zip.ZipInputStream.readEnd(ZipInputStream.java:421)
	at java.util.zip.ZipInputStream.read(ZipInputStream.java:211)
	at java.util.zip.ZipInputStream.closeEntry(ZipInputStream.java:148)
	at java.util.zip.ZipInputStream.getNextEntry(ZipInputStream.java:121)
	at com.absinthe.libchecker.ar0.g(SourceFile:311)
	at com.absinthe.libchecker.ar0.f(SourceFile:1)
	at com.absinthe.libchecker.ar0.b(SourceFile:44)
	at com.absinthe.libchecker.wq0.call(SourceFile:342)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
	at java.lang.Thread.run(Thread.java:1012)
```

**Comment by AyakaAgo** [comment]
> v2.3.7 open but statistics tab not working...

[`.ui.main.LibReferenceActivity`](https://github.com/LibChecker/LibChecker/blob/950d2f679cb84e167e2dee9cc139b7eb1a5bb69f/app/src/main/kotlin/com/absinthe/libchecker/ui/main/LibReferenceActivity.kt#L120) still using `anim/lib_reference_spring.json.zip`

**Comment by qhy040404** [comment]
I found that `anim/lib_reference_spring.json.zip` seems to be corrupted. I cannot unzip it successfully.

**Comment by qhy040404** [comment]
Repacked `anim/lib_reference_spring.json.zip`. Released by [2.3.7](https://github.com/LibChecker/LibChecker/releases/tag/2.3.7)

**Comment by xmha97** [comment]
Thanks for v2.3.7
Everything works properly.
